core/store/src/trie/shard_tries.rs

```diff
@@ -209,6 +209,12 @@ impl ShardTries {
         &self.0.state_snapshot
     }
 
+    #[tracing::instrument(
+        level = "debug",
+        target = "trie",
```

Comment:
Lets go for `store::trie` as the target for consistency with other spans I added around the area (it also allows for sorta-namespace-like filter definitions) and also a `trace` level.